### ISSUE TITLE
react-redux must to be added

### ISSUE THREAD

**Issue by gaoxiaosong** [magicismight/react-native-root-siblings#40]
When I use this package in react native, it has error with following message:

```
error: bundling failed: Error: Unable to resolve module `react-redux` from
`....../node_modules/react-native-root-siblings/index.js`:
Module `react-redux` does not exist in the Haste module map
```

And I found there is `require('react-redux')` in `index.js`. It also need us to add react-redux in package.json~

**Comment by sunnylqm** [comment]
Should be fixed in 3.1.6

**Comment by gaoxiaosong** [comment]
It is OK now.
